app/controllers/JourneyAnswersController.scala

```diff
@@ -71,10 +71,17 @@ class JourneyAnswersController @Inject() (auth: AuthorisedAction,
 
   def saveExpensesTailoringTotalAmountAnswers(taxYear: TaxYear, businessId: BusinessId, nino: Nino): Action[AnyContent] = auth.async {
```

Comment:
Why do we need this method as well as `saveExpensesTailoringCategoryTypeAnswer`? Will we ever be saving the expenses category without the amount?

Reply:
Yes, if answer is NoExpenses we save expensesCategory, if the answer is TotalAmount we need to send totalAmount to API and expensesCategory to DB

Reply:
Ok, but could the totalAmount not be optional then?

Reply:
Ok. I would probably have a single model with it being optional in that case, but maybe a personal preference thing.